File: src/main/java/ru/arty_bikini/crm/dto/orders/DesignDTO.java
package ru.arty_bikini.crm.dto.orders;

import ru.arty_bikini.crm.data.UserEntity;
import ru.arty_bikini.crm.data.enums.DesignStraps;
import ru.arty_bikini.crm.data.enums.GlueDesign;
import ru.arty_bikini.crm.data.enums.TextileOrder;
import ru.arty_bikini.crm.dto.UserDTO;

import javax.persistence.*;

//класс данных о дизайне
public class DesignDTO {
    private String color;//цвет купальника
    private TextileOrder textile;//ткань купальника

    private String amount;//количество страз
    private GlueDesign glue;//клей
    private DesignStraps straps;//верхние лямки дизайн
    private String commentDesignUP;//комментарий по дизайну лифа
    private String commentDesignDoun;//комментарий по дизайну низ

    private UserDTO users;//кто заполнил дизайн
    private boolean userTanya;//таня проверила дизайн

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public TextileOrder getTextile() {
        return textile;
    }

    public void setTextile(TextileOrder textile) {
        this.textile = textile;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public GlueDesign getGlue() {
        return glue;
    }

    public void setGlue(GlueDesign glue) {
        this.glue = glue;
    }

    public DesignStraps getStraps() {
        return straps;
    }

    public void setStraps(DesignStraps straps) {
        this.straps = straps;
    }

    public String getCommentDesignUP() {
        return commentDesignUP;
    }

    public void setCommentDesignUP(String commentDesignUP) {
        this.commentDesignUP = commentDesignUP;
    }

    public String getCommentDesignDoun() {
        return commentDesignDoun;
    }

    public void setCommentDesignDoun(String commentDesignDoun) {
        this.commentDesignDoun = commentDesignDoun;
    }

    public UserDTO getUsers() {
        return users;
    }

    public void setUsers(UserDTO users) {
        this.users = users;
    }

    public boolean isUserTanya() {
        return userTanya;
    }

    public void setUserTanya(boolean userTanya) {
        this.userTanya = userTanya;
    }
}

File: common/src/commonMain/kotlin/ru/arty_bikini/crm/dto/orders/DesignDTO.kt
package ru.arty_bikini.crm.dto.orders

import kotlinx.serialization.Serializable
import ru.arty_bikini.crm.dto.UserDTO
import ru.arty_bikini.crm.dto.enums.design.DesignStraps
import ru.arty_bikini.crm.dto.enums.design.GlueDesign
import ru.arty_bikini.crm.dto.enums.design.TextileOrder

//класс данных о дизайне
@Serializable
class DesignDTO {
    var color: String? = null  //цвет купальника
    var textile: TextileOrder? = null  //ткань купальника
    var amount: String? = null  //количество страз
    var glue: GlueDesign? = null //клей
    var straps: DesignStraps? = null //верхние лямки дизайн
    var commentDesignUP: String? = null  //комментарий по дизайну лифа
    var commentDesignDoun: String? = null  //комментарий по дизайну низ
    var users: UserDTO? = null  //кто заполнил дизайн
    var isUserTanya = false  //таня проверила дизайн
}